bin/kafka-server-stop.sh

```diff
@@ -19,6 +19,6 @@ if [ -z "$PIDS" ]; then
   echo "No kafka server to stop"
   exit 1
 else 
-  kill -SIGTERM $PIDS
+  kill -s TERM $PIDS
```

Comment:
These both match the posix spec as I understand it, and work properly when I run on mac and ubuntu.

Small observation however - since `kill` sends sigterm by default, can we just remove `-s TERM` from these commands?

i.e.
`kill $PIDS`